classes/range/RangeWeight.php

```diff
@@ -85,9 +96,11 @@ public function add($autodate = true, $null_values = false)
     }
 
     /**
-     * Get all available price ranges.
+     * Get all available price weight.
+     *
+     * @param int $id_carrier Carrier identifier
      *
-     * @return array Ranges
+     * @return array|false All range for this carrier
```

Comment:
```suggestion
     * @return array|bool All range for this carrier
```

or do the same for rangeExist & isOverlapping

Reply:
Done